<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>megnelink</class>
 <widget class="QWidget" name="megnelink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>磁力链接转torrent文件</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="file_path_btn">
     <property name="text">
      <string>选择下载路径</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="confirm_btn">
     <property name="text">
      <string>确认转换</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="magnetlink_str">
     <property name="whatsThis">
      <string>输入磁力链接</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>输入磁力链接</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
